### PR TITLE
New module: COPTR/MERGE

### PR DESCRIPTION
**Pull request by ramirobarrantes**
## PR checklist

- [ ] This PR add a module to the coptr pipeline: merging of bam files mapped by coptr.

### CODE REVIEW

File: modules/nf-core/coptr/merge/meta.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "coptr_merge"
+description: Merge reads that were mapped to multiple indeces
```
Comment (itrujnara) [inline modules/nf-core/coptr/merge/meta.yml:4]:
```suggestion
description: Merge reads that were mapped to multiple indices
```

File: modules/nf-core/coptr/merge/meta.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - bams:
+        type: file
+        description: bam files to merge. Assumes that the reads were mapped to different indeces.
```
Comment (itrujnara) [inline modules/nf-core/coptr/merge/meta.yml:29]:
```suggestion
        description: bam files to merge. Assumes that the reads were mapped to different indices.
```

**Review by itrujnara — COMMENTED** [review]
Great job! I've found a couple of typos, otherwise good to go

**Comment by ramirobarrantes** [comment]
> Great job! I've found a couple of typos, otherwise good to go

Thank you. Just did the updates.

**Review by itrujnara — APPROVED** [review]
Good to go once the CI starts to work

**Pull request merged**